apollo-federation/src/correctness/mod.rs

```diff
@@ -73,6 +73,9 @@ pub fn check_plan(
                 "Failed to compute the response shape from query plan:\n{e}"
             ))
         })?;
+    println!(
+        "check_plan:\nOperation response shape: {op_rs}\nQuery plan response shape: {plan_rs}"
+    );
```

Comment:
This is not needed. When you set `RUST_LOG=debug` the tracing::debug invocation below is sent to stdout.

For example:
```
$ RUST_LOG=debug cargo run -p apollo-federation-cli --release -- plan /path/to/operation /path/to/schema

DEBUG check_plan:
Operation response shape: {
```

Reply:
yep it's just a leftover from my local debugging stuff (pushed this PR to show the failing correctness checker cases)